diff -rU3 a/makemkvgui/src/main.cpp b/makemkvgui/src/main.cpp
--- a/makemkvgui/src/main.cpp	2020-12-09 17:33:52.000000000 +0000
+++ b/makemkvgui/src/main.cpp	2020-12-10 18:00:40.889517098 +0000
@@ -17,8 +17,10 @@
 
 #include "mainwnd.h"
 
+#if defined(Q_OS_DARWIN) || defined(Q_OS_WIN)
 #define aversion aversion002
 extern "C" const unsigned int __cdecl aversion();
+#endif
 
 #ifdef Q_OS_WIN
 #include <objbase.h>
